helm/hpcc/templates/dali.yaml

```diff
@@ -124,7 +124,7 @@ kind: ConfigMap
 {{- if and (not $sasha.storage.existingClaim) (not $sasha.storage.plane) }}
 {{ include "hpcc.addPVC" (dict "root" $ "name" (printf "sasha-%s" $sashaName) "me" $sasha.storage) }}
 {{- end }}
-{{- include "hpcc.addPVCsFromPlanes" (dict "root" $ "label" "sasha") }}
+{{- include "hpcc.addPVCsFromPlanes" (dict "root" $ "label" "sasha" "name" $sasha.storage.plane) }}
```

Comment:
does this to check $sasha.storage is valid first (and default to empty dict to protect)

Reply:
It does - in line 123.

Reply:
ok, missed that, thanks